CMakeLists.txt

```diff
@@ -157,9 +163,9 @@ separate build directory. Note: Please remove autogenerated file \
 `CMakeCache.txt` and directory `CMakeFiles` in the current directory.")
```

Comment:
Everywhere you should just use `SLEEF_SOURCE_DIR` and `SLEEF_BINARY_DIR` since those are the standard and expected names. The `project` command defines them and the `VERSION` variables.

https://cmake.org/cmake/help/v3.18/command/project.html#project